src/TestTask/components/AnimatedButton/AnimatedButton.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
```

Comment:
end next one copied

Reply:
Noted. Please see my response to this comment: https://github.com/invariant-labs/webapp/pull/297#discussion_r882119874